### PR TITLE
magicsock: STUN regularly

### PR DESCRIPTION
**Pull request by bradfitz**

### CODE REVIEW

File: version/prop.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package version
```
Comment (crawshaw) [inline version/prop.go]: pkg docs? :)

File: version/prop.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+import "runtime"
+
+// IsMobile reports whether this is a mobile client build.
+func IsMobile() bool {
```
Comment (crawshaw) [inline version/prop.go]: TODO to use this in controlclient?

File: version/prop.go (hunk 1 above)
Comment (bradfitz) [reply version/prop.go]: It already has package docs in the other files (that are `!build redo` and `+build !redo`). This is the portable file.

File: version/prop.go (hunk 2 above)
Comment (bradfitz) [reply version/prop.go]: I'll break this up.

**Comment by bradfitz** [comment]
Merged manually.

**Pull request closed without merging**